src/index.js

```diff
@@ -18,13 +18,15 @@ import App from "./components/App";
 const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
 export const store = createStore(rootReducer, composeEnhancers(applyMiddleware(thunkMiddleware)));
 
+const BENTO_URL_WITH_TRAILING_SLASH = BENTO_URL.endsWith("/") ? BENTO_URL : `${BENTO_URL}/`;
```

Comment:
fails for me on Bento2.8, BENTO_URL is null here so `endsWith()` crashes

Reply:
will fix, ty!

Reply:
should be fixed now